avro/src/main/java/com/fasterxml/jackson/dataformat/avro/AvroMapper.java

```diff
@@ -28,12 +47,13 @@ public AvroMapper() {
     public AvroMapper(AvroFactory f) {
         super(f);
         registerModule(new AvroModule());
+        setDefaultTyping(new AvroSchemaTypeResolverBuilder());
```

Comment:
I am not sure it is a good idea to force default typing just because Avro encoding is used? Although granted use of Jackson's standard polymorphic typing probably won't work well with most Avro type definitions...